docs/using-scylla/tracing.rst

```diff
@@ -9,8 +9,12 @@ Tracing is a ScyllaDB tool meant to help debugging and analyzing internal flows
 * **Probalistic Tracing** randomly chooses a request to be traced with some defined probability.
 * **Slow Query Logging** - records queries with handling time above the specified threshold.
 
-.. note:: If you're planning to use either **probabilistic tracing** or **slow query logging** (see below), it's advisable to change the ``replication_factor`` of the  ``system_traces`` keyspace to ``ALL``. 
+.. note:: If you're planning to use either **probabilistic tracing** or **slow query logging** (see below), it's advisable to change the ``replication_factor`` of the  ``system_traces`` keyspace to ``ALL``:
+          
+          If you use ``NetworkTopologyStrategy``, the replication factor set for each datacenter should equal the number 
```

Comment:
language: "...should BE equal..."

And I'd recommend to use "recommended" instead of "should":

"If you use ``NetworkTopologyStrategy``, we recommend to use a replication factor set to be equal the number..."